assets/src/components/v2/pre_fare/no_data.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ComponentType } from "react";
+import { default as LcdNoData } from "Components/v2/lcd/no_data";
+
+interface Props {
+  show_alternatives: boolean;
+}
+
+const NoData: ComponentType<Props> = ({
+  show_alternatives: showAlternatives,
+}) => {
+  return (
+    <>
+      <div className="no-data-left">
+        <LcdNoData show_alternatives={showAlternatives} />{" "}
```

Comment:
Is the rendered space character (`{" "}`) necessary for some reason? Here and below.

Reply:
Must have been autocomplete or something. Removed those.